<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForwardEngineerSchemaDlg</class>
 <widget class="QDialog" name="ForwardEngineerSchemaDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Schema Object Cypher Statements</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>141</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>141</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select how to generate the cypher statements</string>
     </property>
     <property name="title">
      <string>Generate Options</string>
     </property>
     <widget class="QRadioButton" name="rbDropFirst">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Drops first, then Creates</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbCreateFirst">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Creates first, then Drops</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbDropCreate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>321</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>For each object, Drop then Create</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbDropOnly">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Drops Only</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbCreateOnly">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Creates Only</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>121</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>121</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select which objects to generate</string>
     </property>
     <property name="title">
      <string>Schema Objects</string>
     </property>
     <widget class="QCheckBox" name="cbNodeKey">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Node Key Constraint</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbNodePropertyUnique">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Node Property Unique Constraint</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbNodePropertyExists">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Node Property Exists Constraint</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbRelPropertyExists">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>281</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Relationship Property Exists Constraint</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbIndexAll">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>All Indexes</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbIndexNoConstraint">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Non-Constraint Indexes</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbIndex">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Index</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>43</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>43</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>211</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnGenerate">
        <property name="toolTip">
         <string>Press to generate the Cypher file</string>
        </property>
        <property name="text">
         <string>Generate Cypher File</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>211</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="toolTip">
      <string>Press Close to close the dialog box.</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbDropFirst</tabstop>
  <tabstop>rbCreateFirst</tabstop>
  <tabstop>rbDropCreate</tabstop>
  <tabstop>rbDropOnly</tabstop>
  <tabstop>rbCreateOnly</tabstop>
  <tabstop>cbNodeKey</tabstop>
  <tabstop>cbNodePropertyUnique</tabstop>
  <tabstop>cbNodePropertyExists</tabstop>
  <tabstop>cbRelPropertyExists</tabstop>
  <tabstop>cbIndex</tabstop>
  <tabstop>rbIndexAll</tabstop>
  <tabstop>rbIndexNoConstraint</tabstop>
  <tabstop>btnGenerate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ForwardEngineerSchemaDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ForwardEngineerSchemaDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
